### DISCUSSION TITLE
大佬们，开发个关键字屏蔽模块吧！！！

### DISCUSSION THREAD

**Discussion by Overyet** [the1812/Bilibili-Evolved#3009] [功能建议 / Ideas]
我快让马士兵给烦死了！
看了一些关于语言学习的视频，马士兵就霸占推荐栏了！拉了黑名单。可是招架不住下崽儿呀！！！全是马士兵小马  ，王牌讲师马士兵，马士兵教育............  愁死了

**Comment by the1812** [comment]
你可以先用 `隐藏视频推荐` 功能凑合下

**Comment by Skylarzz** [comment]
可以使用搜索酱凑合下，支持任意网站的关键词屏蔽 [>>](https://meta.appinn.net/t/topic/36029)

**Comment by miozus** [comment]
Java 之父马士兵

**Comment by 233YUZI** [comment]
油猴脚本“哔哩哔哩 - 屏蔽指定内容”https://greasyfork.org/zh-CN/scripts/397669
你的需求完美解决
